src/components/FAQSection.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Accordion, Card } from 'react-bootstrap';
+
+const styles = {
```

Comment:
I think this is still better off in a css file, you don't need to make a new one though feel free to pop it in like index.css or something